### ISSUE TITLE
Export/import commands history list for Quick Launch dialog

### ISSUE THREAD

**Issue by MaxRis** [jumptrading/FluentTerminal#229]

**Comment by peske** [comment]
It doesn't make sense in my opinion. Command history should be a simple transparent auto-complete feature which doesn't require user to engage / maintain / configure it.

To put simply: vast majority of applications have some kind of auto-complete feature for some input, but none of the application offers "export auto-complete history".

Also I don't see any benefit / use-case for such feature.

**Comment by MaxRis** [comment]
There are at least 2 benefits:

- Possibility to edit history exported into JSON format and import it back.
- User can import commands to another machine ( that, probably, also might be done via application settings data saved as shared via online MS account )

**Comment by mjs** [comment]
I have to agree with @peske here. I don't think this is high priority.

Did this come from a particular user need or problem?

**Comment by MaxRis** [comment]
Yes, Chris asked for possibility to edit all commands in history at once to do switch from ConPty to WinPty.

**Comment by MaxRis** [comment]
That's related to https://github.com/jumptrading/FluentTerminal/issues/230

**Comment by cjw296** [comment]
Sure, my question was mainly around a workaround for another issue. I'd be totally fine with this issue being closed completely.
